share/completions/mysql.fish

```diff
@@ -1,3 +1,5 @@
+__fish_complete_mysql mysql
```

Comment:
Just put the function in here. It's not used anywhere else, so this indirection isn't helpful.

Reply:
See https://github.com/fish-shell/fish-shell/pull/7397#discussion_r504672074